src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -42,12 +48,15 @@ protected string ParseSecretToken(ConfigurationKeyValue kv)
 
 		protected LogLevel ParseLogLevel(ConfigurationKeyValue kv)
 		{
-			if (kv == null || string.IsNullOrEmpty(kv.Value)) return ConsoleLogger.DefaultLogLevel;
-
-			if (Enum.TryParse<LogLevel>(kv.Value, out var logLevel)) return logLevel;
+			if (TryParseLogLevel(kv?.Value, out var level)) return level.Value;
 
-			Logger?.LogError("Failed parsing log level from {Origin}: {Key}, value: {Value}. Defaulting to log level '{DefaultLogLevel}'",
-				kv.ReadFrom, kv.Key, kv.Value, ConsoleLogger.DefaultLogLevel);
+			if (kv?.Value == null)
+				Logger?.LogDebug("No log level provided. Defaulting to log level '{DefaultLogLevel}'", ConsoleLogger.DefaultLogLevel);
```

Comment:
This is a chicken-egg (🐔 🥚)  problem 🤣  We print with `LogDebug`, but set the `LogLevel` to `LogLevel.Error`, so this will never show up in the logs. One solution that comes to my mind is that we default to `LogLevel.Information` and print it as `LogInfo`. This is not high prio...